index.bs

```diff
@@ -398,6 +401,8 @@ Otherwise, do nothing.
 <emu-alg>
   1. Let _called_ be the number of times _count_ has been invoked (including this invocation) with
```

Comment:
I might replace this line with the the two you added, since they are a more explicit explanation of what was originally here (this line).